<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>757</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>402</width>
    <height>757</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>쿠쿠이츠</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="maximumSize">
       <size>
        <width>380</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <widget class="QPushButton" name="start">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>-10</y>
          <width>380</width>
          <height>711</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    border: none;
    background-color: transparent;
}
</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="mainIcon.qrc">
          <normaloff>:/mainIcon/StartIMG.png</normaloff>:/mainIcon/StartIMG.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>400</width>
          <height>711</height>
         </size>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_main">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QGridLayout" name="layout_top_pin" columnstretch="5,1,0">
            <item row="0" column="2">
             <widget class="QPushButton" name="btn_alarm">
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="mainIcon.qrc">
                <normaloff>:/mainIcon/alarm.png</normaloff>:/mainIcon/alarm.png</iconset>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QPushButton" name="btn_top_addr">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>광주인력개발원</string>
              </property>
              <property name="icon">
               <iconset resource="mainIcon.qrc">
                <normaloff>:/mainIcon/ic_map_pin.png</normaloff>:/mainIcon/ic_map_pin.png</iconset>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QScrollArea" name="main_scroll">
            <property name="minimumSize">
             <size>
              <width>370</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>380</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>374</width>
               <height>739</height>
              </rect>
             </property>
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="1" column="0">
               <layout class="QGridLayout" name="gridLayout_2">
                <item row="2" column="0">
                 <widget class="QLabel" name="wow_event_img">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>350</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="pixmap">
                   <pixmap resource="mainIcon.qrc">:/mainIcon/wow_event.jpg</pixmap>
                  </property>
                  <property name="scaledContents">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <layout class="QGridLayout" name="layout_category"/>
                </item>
                <item row="7" column="0">
                 <widget class="QScrollArea" name="h_scroll_category">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>80</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>345</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="widgetResizable">
                   <bool>true</bool>
                  </property>
                  <widget class="QWidget" name="scrollAreaWidgetContents_3">
                   <property name="geometry">
                    <rect>
                     <x>0</x>
                     <y>0</y>
                     <width>343</width>
                     <height>78</height>
                    </rect>
                   </property>
                   <layout class="QGridLayout" name="gridLayout_6"/>
                  </widget>
                 </widget>
                </item>
                <item row="6" column="0">
                 <widget class="QLabel" name="label_4">
                  <property name="font">
                   <font>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>골라먹는 맛집</string>
                  </property>
                 </widget>
                </item>
                <item row="8" column="0">
                 <layout class="QHBoxLayout" name="check_layout" stretch="0,0,1,1,1">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QCheckBox" name="checkBox">
                    <property name="text">
                     <string/>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButton">
                    <property name="maximumSize">
                     <size>
                      <width>30</width>
                      <height>30</height>
                     </size>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                    <property name="icon">
                     <iconset resource="mainIcon.qrc">
                      <normaloff>:/mainIcon/wow_img.png</normaloff>:/mainIcon/wow_img.png</iconset>
                    </property>
                    <property name="iconSize">
                     <size>
                      <width>30</width>
                      <height>30</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_6">
                    <property name="font">
                     <font>
                      <bold>true</bold>
                     </font>
                    </property>
                    <property name="text">
                     <string>와우할인</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QComboBox" name="comboBox">
                    <item>
                     <property name="text">
                      <string>추천순</string>
                     </property>
                    </item>
                   </widget>
                  </item>
                  <item>
                   <widget class="QComboBox" name="comboBox_2">
                    <item>
                     <property name="text">
                      <string>필터</string>
                     </property>
                    </item>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item row="9" column="0">
                 <layout class="QVBoxLayout" name="layout_last"/>
                </item>
                <item row="3" column="0">
                 <layout class="QHBoxLayout" name="horizontalLayout" stretch="5,1,0">
                  <item>
                   <widget class="QLabel" name="label_2">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="font">
                     <font>
                      <bold>true</bold>
                     </font>
                    </property>
                    <property name="text">
                     <string>이츠 추천 맛집</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer">
                    <property name="orientation">
                     <enum>Qt::Orientation::Horizontal</enum>
                    </property>
                    <property name="sizeType">
                     <enum>QSizePolicy::Policy::Fixed</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>220</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QPushButton" name="btn_right">
                    <property name="maximumSize">
                     <size>
                      <width>30</width>
                      <height>30</height>
                     </size>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                    <property name="icon">
                     <iconset resource="mainIcon.qrc">
                      <normaloff>:/mainIcon/right.jpg</normaloff>:/mainIcon/right.jpg</iconset>
                    </property>
                    <property name="iconSize">
                     <size>
                      <width>24</width>
                      <height>24</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item row="0" column="0">
                 <widget class="QPushButton" name="btn_search">
                  <property name="styleSheet">
                   <string notr="true"/>
                  </property>
                  <property name="text">
                   <string> 검색</string>
                  </property>
                  <property name="icon">
                   <iconset resource="mainIcon.qrc">
                    <normaloff>:/mainIcon/serch.png</normaloff>:/mainIcon/serch.png</iconset>
                  </property>
                 </widget>
                </item>
                <item row="4" column="0">
                 <widget class="QScrollArea" name="scrollArea_side_slide">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                    <horstretch>125</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>290</width>
                    <height>200</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>345</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="verticalScrollBarPolicy">
                   <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
                  </property>
                  <property name="horizontalScrollBarPolicy">
                   <enum>Qt::ScrollBarPolicy::ScrollBarAsNeeded</enum>
                  </property>
                  <property name="widgetResizable">
                   <bool>false</bool>
                  </property>
                  <widget class="QWidget" name="scrollAreaWidgetContents_2">
                   <property name="geometry">
                    <rect>
                     <x>0</x>
                     <y>0</y>
                     <width>326</width>
                     <height>158</height>
                    </rect>
                   </property>
                   <layout class="QGridLayout" name="gridLayout_5"/>
                  </widget>
                 </widget>
                </item>
                <item row="5" column="0">
                 <widget class="QStackedWidget" name="ad_stacked">
                  <property name="minimumSize">
                   <size>
                    <width>150</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="currentIndex">
                   <number>0</number>
                  </property>
                  <widget class="QWidget" name="page_2">
                   <layout class="QGridLayout" name="gridLayout_8">
                    <item row="0" column="0">
                     <widget class="QLabel" name="label">
                      <property name="maximumSize">
                       <size>
                        <width>16777215</width>
                        <height>150</height>
                       </size>
                      </property>
                      <property name="text">
                       <string/>
                      </property>
                      <property name="pixmap">
                       <pixmap resource="mainIcon.qrc">:/mainIcon/advertising.jpg</pixmap>
                      </property>
                      <property name="scaledContents">
                       <bool>true</bool>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                  <widget class="QWidget" name="page_3">
                   <layout class="QGridLayout" name="gridLayout_9">
                    <item row="0" column="0">
                     <widget class="QLabel" name="label_3">
                      <property name="text">
                       <string/>
                      </property>
                      <property name="pixmap">
                       <pixmap resource="mainIcon.qrc">:/mainIcon/resize_maru.png</pixmap>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                  <widget class="QWidget" name="page_4">
                   <layout class="QGridLayout" name="gridLayout_10">
                    <item row="0" column="0">
                     <widget class="QLabel" name="label_5">
                      <property name="text">
                       <string/>
                      </property>
                      <property name="pixmap">
                       <pixmap resource="mainIcon.qrc">:/mainIcon/resize_ttoek.png</pixmap>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                  <widget class="QWidget" name="page_5">
                   <layout class="QGridLayout" name="gridLayout_11">
                    <item row="0" column="0">
                     <widget class="QLabel" name="label_7">
                      <property name="text">
                       <string/>
                      </property>
                      <property name="pixmap">
                       <pixmap resource="mainIcon.qrc">:/mainIcon/resize_star.png</pixmap>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                  <widget class="QWidget" name="page_6">
                   <layout class="QGridLayout" name="gridLayout_12">
                    <item row="0" column="0">
                     <widget class="QLabel" name="label_8">
                      <property name="text">
                       <string/>
                      </property>
                      <property name="pixmap">
                       <pixmap resource="mainIcon.qrc">:/mainIcon/resize_hosik.png</pixmap>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="BtnHome">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="mainIcon.qrc">
              <normaloff>:/mainIcon/uniform_icon_2.png</normaloff>:/mainIcon/uniform_icon_2.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>64</width>
              <height>64</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="BtnUnderSearch">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="mainIcon.qrc">
              <normaloff>:/mainIcon/uniform_icon_5.png</normaloff>:/mainIcon/uniform_icon_5.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>64</width>
              <height>64</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="BtnFavorite">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="mainIcon.qrc">
              <normaloff>:/mainIcon/uniform_icon_1.png</normaloff>:/mainIcon/uniform_icon_1.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>64</width>
              <height>64</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="BtnOrder">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="mainIcon.qrc">
              <normaloff>:/mainIcon/uniform_icon_4.png</normaloff>:/mainIcon/uniform_icon_4.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>64</width>
              <height>64</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="BtnMyEats">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="mainIcon.qrc">
              <normaloff>:/mainIcon/uniform_icon_3.png</normaloff>:/mainIcon/uniform_icon_3.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>64</width>
              <height>64</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QFrame" name="cart_frame">
          <property name="maximumSize">
           <size>
            <width>360</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::Shape::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Shadow::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="0">
            <widget class="QPushButton" name="btn_show_cart_page">
             <property name="minimumSize">
              <size>
               <width>340</width>
               <height>0</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
        background-color: #00A9FF;
        color: white;
        font-weight: bold;
        border-radius: 8px;
        padding: 10px 20px;
    }
    QPushButton:hover {
        background-color: #0088CC;
    }

</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="mainIcon.qrc"/>
 </resources>
 <connections/>
</ui>
